hedera-node/hedera-consensus-service-impl/src/main/java/module-info.java

```diff
@@ -6,6 +6,7 @@
     requires transitive com.hedera.node.app.spi;
     requires transitive com.hedera.node.hapi;
     requires transitive com.hedera.pbj.runtime;
+    requires transitive com.swirlds.merkle;
```

Comment:
Is this intentionally here? I'm trying to figure out how this relates to the rest of the changes

Reply:
@mhess-swl Yes, it is intentional. It resolves the failing dependency check report when running the `./gradlew checkAllModuleInfo` command.